README.md

```diff
@@ -46,14 +45,45 @@ By default `ON`, set it to `OFF` if you do not want to update the already existi
 
 Required for installing. Defines the path where the library is to be installed. Set it to, `/usr/local/include` on Linux based systems.
 
+**Windows**
+
+1. git clone https://github.com/codezonediitj/adaboost
+2. Move to back to parent directory, `cd ../`
+3. Execute, `mkdir build-adaboost`
+4. Execute, `cd build-adaboost`
+5. Install CMake from - https://cmake.org/download/
+- After downloading locate the bin folder under CMake directory and copy it's address(Like if the folder is located in C drive's Program files,address can be -C:\Program Files\cmake\bin).
+- In the search bar on task bar type environment variables. Then click on the `Edit the System Variables`.
+- The system properties dialog box will open, click on `Environment Variables`.
+- Click on `Path` under system variables window. Then click `Edit` under the same.
+- The edit enviornment dialog box will open, Click `New` and add the copied address of bin folder. Then Click `ok`.
+- Similarly add the paths to your compilers (for example - if   it's MinGW add path to its bin folder for instance C:\MinGW\bin)
```

Comment:
Does this work? You told in chat that, `C:\MinGW` threw parser error. Should we use `/` or `\` in path for making things work.

Reply:
Yes, these paths need to be added to the environment variables in the control panel, the error I showed you was different from this.

Reply:
In windows , `\` is used for addressing, so it works.